site/content/en/references/kustomize/kustomization/patches/_index.md

```diff
@@ -17,12 +17,12 @@ The `patches` field contains a list of patches to be applied in the order they a
 
 Each patch may:
 
-- be either a [strategic merge] patch, or a [JSON] patch
+- be either a [strategic merge] patch, or a [JSON6902] patch
 - be either a file, or an inline string
 - target a single resource or multiple resources
 
-The patch target selects resources by group, version, kind, name, namespace, labelSelector and
-annotationSelector. Any resource which matches all the **specified** fields has the patch applied
+The patch target selects resources by `group`, `version`, `kind`, `name`, `namespace`, `labelSelector` and
+`annotationSelector`. Any resource which matches all the **specified** fields has the patch applied
 to it (regular expressions). 
 
 A patch can refer to a resource by any of its previous names or kinds.
```

Comment:
Let's move this paragraph to right above `##Examples`

Reply:
Thank you for both changes, it make a lot more sense to group everything that way :)